### ISSUE TITLE
Error on https://goliat.neocities.org/search/msn/music.html

### ISSUE THREAD

**Issue by webunderground** [ruffle-rs/ruffle#8893]
# Error Info
Error name: TypeError
Error message: __webpack_modules__[moduleId] is not a function
Error stack:
```
TypeError: __webpack_modules__[moduleId] is not a function
    at Function.__webpack_require__ (chrome-extension://donbcfbmhbcapadipfkeojnmajbakjdc/dist/ruffle.js?id=12568288548:39:41)
    at b (https://web.archive.org/web/20080818123531js_/http://video.msn.com/ExternalControls/GalleryWidget/v1/script/gallerywidgetscript.js:13:5068)
    at async fetchRuffle (chrome-extension://donbcfbmhbcapadipfkeojnmajbakjdc/dist/ruffle.js?id=12568288548:509:39)
```

# Player Info
Allows script access: true

# Page Info
Page URL: https://goliat.neocities.org/search/msn/music.html

# Browser Info
User Agent: Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36
Platform: Win32
Has touch support: false

# Ruffle Info
Version: 0.1.0
Name: nightly 2022-12-22
Channel: nightly
Built: 2022-12-22T00:14:42.760Z
Commit: 7a09dd563980b378eb2b6702a227c7b346b5ae3e
Is extension: true

# Metadata

**Comment by n0samu** [comment]
This error occurs because a script on the page (`https://web.archive.org/web/20080818123531js_/http://video.msn.com/ExternalControls/GalleryWidget/v1/script/gallerywidgetscript.js`) overrides `Function.prototype.bind` with its own problematic implementation. Blocking that script resolves the issue.

But even after the issue is resolved, the Flash content fails to load with a different error, because the SWF URL points directly to Wayback Machine, and the request is blocked by CORS. If the webmaster wants the Flash content to work correctly, they will also need to rehost the SWF file on their site. [See the wiki for more information](https://github.com/ruffle-rs/ruffle/wiki/Using-Ruffle#configure-cors-header), particularly the last bullet point.

**Comment by Lord-McSweeney** [comment]
If you go to the SWF URL directly, it's still broken (renders a solid grey rectangle and another one with a gradient), although no warnings or errors are logged.

**Comment by n0samu** [comment]
Yes, and Adobe Flash Player shows the same. So not a Ruffle issue.
